client/app/views/admin/landing.html

```diff
@@ -2,40 +2,54 @@
-          <li>
-            <span data-translate>You can reach GlobaLeaks's support chat via IRC on OFTC server on channel #globaleaks:</span> <a href="https://webchat.oftc.net/?channels=globaleaks" target="_blank" rel="noreferrer noopener"  data-translate>GlobaLeaks Support Chat</a>
-          </li>
-          <li>
-            <span data-translate>To properly organize your project, please read the following guidelines:</span> <a href="https://docs.google.com/document/d/1Y5h_lSZq-MsefH1LkEjHN5uOfEccyyI4oJgd7tm2Iho/pub" target="_blank" rel="noreferrer noopener"  data-translate>Guidelines on how to start up Whistleblowing Projects</a>
-          </li>
-          <li>
-            <span data-translate>To report a bug or request a feature in GlobaLeaks software, please open a ticket describing the issue you're encountering or the new feature request:</span> <a href="https://github.com/globaleaks/GlobaLeaks/issues" target="_blank" rel="noreferrer noopener"  data-translate>GlobaLeaks Ticketing System</a>
-          </li>
-          <li>
-            <span data-translate>Remember to provide proper security advice to Whistleblowers:</span> <a href="https://docs.google.com/document/d/1ZrndvBj9eTg-ooIRfKbXxX18Ie-ODlcjnHjKXSY78Ew/edit?usp=sharing" target="_blank" rel="noreferrer noopener" data-translate>Security advice for whistleblowers</a>
-          </li>
-          <li>
-            <span data-translate>To know more about GlobaLeaks Security you can read the following document:</span> <a href="https://github.com/globaleaks/globaleaks/wiki#software-security" target="_blank" rel="noreferrer noopener"  data-translate>GlobaLeaks Security</a>
-          </li>
-        </ul>
-        <div class="title3" data-translate>Contacts</div>
-        <ul>
-          <li>
-            <span data-translate>If you want to tell us more about your project, or you need some help, write us at:</span> <a href="mailto:projects@hermescenter.org">projects@hermescenter.org</a>
-          </li>
-          <li>
-            <span data-translate>For technical support inquiries you can write us at:</span> <a href="mailto:support@hermescenter.org">support@hermescenter.org</a>
-          </li>
-          <li>
-            <span data-translate>Follow us on Twitter if you like to be profiled by NSA:</span> <a target="_blank" rel="noreferrer noopener"  href="https://twitter.com/globaleaks">https://twitter.com/globaleaks</a>
-          </li>
-        </ul>
+      <div data-ng-controller="AdminAboutCtrl">
+        <uib-tabset>
+          <uib-tab data-index="0" heading="{{'Home' | translate}}">
+            <div class="tab-content" data-ng-include="'views/admin/about/contact.html'"></div>
+          </uib-tab>
+
+          <uib-tab data-index="1">
+            <uib-tab-heading>
+              <span data-translate>Change log</span>
+            </uib-tab-heading>
+            <div class="tab-content">
+              <div class="btn-group" role="group" aria-label="Release notes toggle.">
```

Comment:
for uniformity reasons let's avoid to start aria labels until propre analisys of what to do in relations to manual aria settings.